E3Next/Processors/Movement.cs

```diff
@@ -74,7 +74,7 @@ public static void Check_Chase()
                                     bool pathExists = MQ.Query<bool>($"${{Navigation.PathExists[id {spawnID}]}}");
                                     if (pathExists)
                                     {
-                                        MQ.Cmd($"/squelch /nav id {spawnID} log=error");
+                                        MQ.Cmd($"/squelch /nav id {spawnID} dist=20 log=error");
```

Comment:
this has nothing to do with command on startup. 
also hard coded values off the normal default behavior will need an option in the general settings or character settings.

Reply:
I Have just removed this change atm as I agree it should be adjustable .. but just a note It currently is hard coded .. and all my people just ping pong around with selos in group as everyone tries to be on top